src/Faker/Provider/Payment.php

```diff
@@ -297,16 +297,22 @@ public static function iban($countryCode = null, $prefix = '', $length = null)
     }
 
     /**
-     * Return the String of a SWIFT/BIC number
+     * Return the String of a SWIFT/BIC number.
      *
      * @example 'RZTIAT22263'
      *
      * @see    http://en.wikipedia.org/wiki/ISO_9362
      *
+     * @param string|null $countryCode ISO 3166-1 alpha-2 country code
+     *
      * @return string Swift/Bic number
      */
-    public static function swiftBicNumber()
+    public static function swiftBicNumber($countryCode = null)
```

Comment:
This is a bit shady since you have no clue what can be put in here and all can be invalid. I think we have a list of all country codes already in the package. Can you use that to verify it exists?